aten/src/ATen/native/metal/mpscnn/MPSCNNFullyConnectedOp.h

```diff
@@ -3,11 +3,10 @@
 #import <ATen/native/metal/mpscnn/MPSCNNConvOp.h>
 #import <Foundation/Foundation.h>
 
-using namespace at::native::metal;
 API_AVAILABLE(ios(11.0), macos(10.13))
 @interface MPSCNNFullyConnectedOp : NSObject<MPSCNNOp>
 + (MPSCNNFullyConnectedOp*)linear:(const Conv2DParams&)params
                           weights:(float*)w
                              bias:(float*)b
                      neuronFilter:(NeuronType)t;
-@end
+@end
```

Comment:
Don't we have a lint about newline at the end of the file?

Reply:
If that's the case I'll revert the diff instead.

Reply:
Well, OSS lint is green, but we need to start enforcing it for `.mm` files at some point